assets/app/scripts/services/alertMessage.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .service("AlertMessageService", function(){
+    var alerts = [];
+    return {
+      addAlert: function(alert) {
```

Comment:
need a way to clear the alerts or once you have added an alert to this its going to show up every time you navigate to the project list